msg/AttentionTargets.msg

```diff
@@ -0,0 +1 @@
+semio_msgs_ros/AttentionTarget[] targets
```

Comment:
Right now, "targets" (specifically in the context of attention) make sense; however, we should consider this as being something like an "object" or "entity" or "thing" in the environment, as this might offer a contextual clue to other things (e.g., things in the environment that the person is talking about or might be pointing at).